src/Main.elm

```diff
@@ -3523,17 +3523,24 @@ viewFormFieldOptionsBuilder shortTextTypeList index formFields formField =
                 ++ [ div [ class "tff-field-group" ]
                         [ label [ class "tff-field-label" ] [ text "Minimum required" ]
                         , input
-                            [ type_ "number"
+                            ([ type_ "number"
                             , class "tff-text-field"
                             , value (minRequired |> Maybe.map String.fromInt |> Maybe.withDefault "")
-                            , Attr.min "0"
+                            , Attr.min (if formField.presence == System then "1" else "0")
```

Comment:
make the choose multiple min input field 1 instead of 0

Reply:
make sure elm-format had run, i.e. `make format`. this 1-liner should auto change to multiple lines